tests/e2e/cluster_edit_test.go

```diff
@@ -329,6 +330,18 @@ var _ = Describe("Edit cluster", ci.Day2, func() {
 			}, "Attribute private, cannot be changed from")
 		})
 
+		It("imdsv2 fields - [id:75414]", ci.Medium, ci.FeatureClusterEc2MetadataHttpTokens, func() {
+			By("Try to edit ec2_metadata_http_tokens value")
+			otherHttpToken := constants.DefaultEc2MetadataHttpTokens
+			if profile.Ec2MetadataHttpTokens == "" ||
+				profile.Ec2MetadataHttpTokens == constants.OptionalEc2MetadataHttpTokens {
+				otherHttpToken = constants.RequiredEc2MetadataHttpTokens
+			}
```

Comment:
Why not this ?
```suggestion
			otherHttpToken := profile.Ec2MetadataHttpTokens
			if otherHttpToken == constants.RequiredEc2MetadataHttpTokens {
				otherHttpToken = constants.OptionalEc2MetadataHttpTokens
			} else {
			        otherHttpToken = constants.RequiredEc2MetadataHttpTokens
			}
```

Reply:
This will not take into account profile.Ec2MetadataHttpTokens == ""